nomad/run-app_install-local.nomad.hcl

```diff
@@ -0,0 +1,35 @@
+job "app_install_scenario" {
+  type = "batch"
+  group "app_install" {
+    task "start_holochain" {
+      lifecycle {
+        hook = "prestart"
+        sidecar = true
+      }
+
+      driver = "raw_exec"
+      config {
+        command = "bash"
+        args = ["-c", "hc s clean && echo 1234 | hc s --piped create && echo 1234 | hc s --piped -f 8888 run"]
+      }
+    }
+
+    task "run_scenario" {
+      driver = "raw_exec"
+      env {
+        RUST_LOG = "info"
+      }
+      config {
+        command = abspath("result/bin/app_install")
+        args = [
+          "--connection-string", "ws://localhost:8888",
+          "--agents", "2",
+          "--behaviour", "minimal:1",
+          "--behaviour", "large:1",
+          "--duration", "5",
```

Comment:
Should this default to a longer time when you're ready to merge this?

Reply:
My idea (already in the latest version) is that if no duration is provided then it is not actually set and so it will use the default provided by the scenario. This could be an issue for neverending scenarios but I think it makes the most sense and we can always kill a running scenario at set the duration if we miss it by accident.